cli_tools/google-osconfig-agent/main.go

```diff
@@ -83,6 +83,10 @@ func main() {
 	flag.Parse()
 	ctx := context.Background()
 
+	if err := config.SetConfig(); err != nil {
```

Comment:
We don't want to do this here, only during service run

Reply:
as per offline discussion, leaving it as it is